src/Microsoft.Data.SqlClient/tests/ManualTests/DataCommon/DataTestUtility.cs

```diff
@@ -180,7 +180,9 @@ static DataTestUtility()
             AliasName = c.AliasName;
             IsJsonSupported = c.IsJsonSupported;
 
+#if NETFRAMEWORK
```

Comment:
Can you elaborate on this change?

Reply:
ServicePointManager is marked as obsolete in Net 9.
https://learn.microsoft.com/en-us/dotnet/fundamentals/syslib-diagnostics/syslib0014
I removed the call to no effect in the tests, so it didn't seem to be changing anything. But if there's any concern, I can add a suppression instead.

Reply:
@mdaigle Are you sure it's the ServicePointManager? I had seen similar warnings in the code when using Tls12 security type.